internal/resources/resource_host.go

```diff
@@ -508,6 +508,7 @@ func resourceMetalHostCreate(d *schema.ResourceData, meta interface{}) (err erro
 		Pending: []string{
 			string(rest.HOSTSTATE_NEW),
 			string(rest.HOSTSTATE_IMAGING_PREP),
+			string(rest.HOSTSTATE_ISCSI_ATTACHING),
```

Comment:
Question...
```go
string(rest.HOSTSTATE_ATTACHING)
```
is only for FC?

Reply:
> In line 514 is
> ```go
> string(rest.HOSTSTATE_ATTACHING)
> ```
> only for FC?

Yes, sort of, it's for all but iSCSI. For now, it's only FC.